### PR TITLE
API-5172-integration-tests

### PR DESCRIPTION
**Pull request by jhulbert-va**
# [API-5172](https://vajira.max.gov/browse/API-5172)

### CODE REVIEW

File: charon-tests/src/main/docker/entrypoint.sh [hunk 1]
```diff
@@ -1,25 +1,49 @@
...
+  if [ -z "${SENTINEL_ENV:-}" ]; then SENTINEL_ENV=$K8S_ENVIRONMENT; fi
+  if [ -z "${CHARON_URL:-}" ]; then CHARON_URL=https://$K8S_LOAD_BALANCER; fi
+  if [ -z "${VISTA_AVAILABLE:-}" ]; then VISTA_AVAILABLE="true"; fi
+}
+
+main() {
+  addToSystemProperties "sentinel" "${SENTINEL_ENV}"
+  addToSystemProperties "sentinel.charon.url" "${CHARON_URL}"
+  addToSystemProperties "client-key" "${CLIENT_KEY}"
+  addToSystemProperties "vista.access-code" "${VISTA_ACCESS_CODE}"
```
Comment (bschofield-va) [inline charon-tests/src/main/docker/entrypoint.sh:25]: I think you want to plan ahead for when support app proxy users.
```
vista.standard-user.access-code
vista.standard-user.verify-code
```
So you have room for
```
vista.application-proxy-user.access-code
vista.application-proxy-user.verify-code
vista.application-proxy-user.name
```

Comment (jhulbert-va) [reply charon-tests/src/main/docker/entrypoint.sh:25]: That was the already existing property name. Do we want to change it as a part of this PR?

File: charon-tests/src/test/java/gov/va/api/lighthouse/charon/tests/BadRpcContextIT.java [hunk 2]
```diff
@@ -15,7 +15,8 @@
   @SneakyThrows
   void requestForbiddenRpcContext() {
     var systemDefinition = SystemDefinitions.get();
-    assumeTrue(systemDefinition.isVistaAvailable());
+    // Ur boi (Joshy-Boi) can't get this to return a 403, only a 200 with an error message
+    assumeTrue(false);
```
Comment (bschofield-va) [inline charon-tests/src/test/java/gov/va/api/lighthouse/charon/tests/BadRpcContextIT.java:19]: What's our plan here? A TODO with a reference to a Jira item?

Comment (jhulbert-va) [reply charon-tests/src/test/java/gov/va/api/lighthouse/charon/tests/BadRpcContextIT.java:19]: That's what I think. I spend like an hour using different combos and wasn't able to get it to 403. We haven't been running the IT's in QA, so I don't know if it ever actually worked. :shrug:

**Review by bschofield-va — DISMISSED** [review]

**Pull request merged**